src/sentry/data_export/models.py

```diff
@@ -128,3 +128,16 @@ class Meta:
         db_table = "sentry_exporteddata"
 
     __repr__ = sane_repr("query_type", "query_info")
+
+
+class ExportedDataBlob(Model):
+    __core__ = False
+
+    data_export = FlexibleForeignKey("sentry.ExportedData")
+    blob = FlexibleForeignKey("sentry.FileBlob")
+    offset = BoundedPositiveIntegerField()
```

Comment:
Is there _any_ situation where we envision this being more than ~2.7B? It's not clear to me what this actuall represents, but if it's bytes, we should 100% use `BoundedBigIntegerField`. We have this issue currently in other files.

Reply:
It is possible, however this value is later used to create a `FileBlobIndex` object which currently also uses `BoundedPositiveIntegerField`. Because of this, there is an explicit check to ensure the offset is less than `MAX_FILE_SIZE`. Is there something else I should be using instead?

Reply:
Yeah, the FileBlobIndex is the problematic one for us. I'd suggest making this field a bigint to future proof it for when we ever upgrade the column on FileBlobIndex. So maybe keep the check in Python, but use a bigint field so we don't have to do an impossible migration later.